### ISSUE TITLE
Uncaught TypeError: Cannot read property 'pull' of null

### ISSUE THREAD

**Issue by robertdxyz** [spark/particle-dev-local-compiler#2]
```
TypeError: Cannot read property 'pull' of null
  at Object.module.exports.ParticleDevLocalCompiler.updateFirmwareVersions (/Users/robert/.particledev/packages/particle-dev-local-compiler/lib/main.coffee:214:17)
  at atom-workspace.<anonymous> (/Users/robert/.particledev/packages/particle-dev-local-compiler/lib/main.coffee:158:47)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Particle Dev.app/Contents/Resources/app.asar/src/command-registry.js:245:29)
  at /Applications/Particle Dev.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/Applications/Particle Dev.app/Contents/Resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:159:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/Applications/Particle Dev.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-div.atom.commands.add.core:confirm (/Applications/Particle Dev.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Particle Dev.app/Contents/Resources/app.asar/src/command-registry.js:245:29)
  at /Applications/Particle Dev.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Particle Dev.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:524:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Particle Dev.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:347:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Particle Dev.app/Contents/Resources/app.asar/src/window-event-handler.js:184:20)
```

**Comment by robertdxyz** [comment]
@suda did I miss to configure something?

**Comment by suda** [comment]
@robertdolca have you entered Docker settings from step 8: https://github.com/spark/particle-dev-local-compiler#install-and-setup-this-package ?

**Comment by robertdxyz** [comment]
Yes. I configured a different docker-machine name in settings. Besides that I didn't set anything. Shodn't it detect those settings using `docker-machine env`?

**Comment by suda** [comment]
In the future yes, but for now you have to fill all info manually.

**Comment by robertdxyz** [comment]
It works after seting that. It would be nice to report a nice error or something.
Thanks

**Comment by suda** [comment]
I'm on it :)

**Comment by suda** [comment]
Fixed in [v0.0.3](https://github.com/spark/particle-dev-local-compiler/releases/tag/v0.0.3) :)
